doc/commands/check-upgrade.8.rst

```diff
@@ -0,0 +1,93 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _check-upgrade_command_ref-label:
+
+######################
+ Check-Upgrade Command
+######################
+
+Synopsis
+========
+
+``dnf5 check-upgrade [options] [<package-spec>...]``
+
+
+Description
+===========
+
+Non-interactively checks for available updates of specified packages. If no ``<package-spec>``
+is provided, it verifies updates for the entire system.
```

Comment:
To "verify updates" sounds very strange to me, so I'd suggest to use "check for" again. I can't think of any synonym that would fit well enough. But it you are sure about the "verify", feel free to keep it.

Reply:
I agree, I also cannot think of anything else better than "check for" :)